molecule/debian_11/molecule.yml

```diff
@@ -0,0 +1,57 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
+platforms:
+  - name: debian11_systemd
+    image: cisagov/docker-debian11-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+  - name: kali_systemd
+    image: cisagov/docker-kali-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      # The OS family and distribution for Kali is "Kali GNU/Linux",
+      # which is not an allowable group name in Ansible due to the
+      # space and the slash.  This option converts those characters to
+      # underscores when creating the group name.
+      force_valid_group_names: always
+scenario:
+  name: debian_11
+  test_sequence:
+    - dependency
+    - lint
+    - cleanup
+    - destroy
+    - syntax
+    - create
+    - prepare
+    - converge
+    # We remove idempotence since aws/efs-utils breaks it.  It does
+    # this by forcing the installation of python2, which causes
+    # Ansible to try to install python-apt during idempotence.  This
+    # package does not exist in Debian 11 (Bullseye).
+    # - idempotence
+    - side_effect
+    - verify
+    - cleanup
+    - destroy
```

Comment:
I saw your issue (https://github.com/aws/efs-utils/issues/57) on the source repository, but I would like an issue for this repository to make a note to undo this once they eventually drop the mandatory installation of Py2.

Reply:
See commit f59e7d9.

Reply:
Thanks 👍